devicetypes/smartthings/zigbee-multi-button.src/zigbee-multi-button.groovy

```diff
@@ -70,11 +70,11 @@ def parseAttrMessage(description) {
 		map = getButtonEvent(descMap)
 	} else if(descMap?.clusterInt == 0xFC80) {
```

Comment:
is this cluster only used by the Heiman devices?

Reply:
yes, no conflict with other devices